### ISSUE TITLE
Do not generate error for filetypes that cannot be uploaded

### ISSUE THREAD

**Issue by thepurpleblob** [turnitin/moodle-plagiarism_turnitin#684]
This happens when an Assignment is configured for a non-document file type but Turnitin is accidentally left on. This happens...

<img width="611" alt="Screenshot 2023-12-12 at 14 04 13" src="https://github.com/turnitin/moodle-plagiarism_turnitin/assets/664139/64a78bee-caf6-4fcd-8630-2d5c3d0f775b">

There is absolute no indication that the error block only relates to Turnitin. So students are, not surprisingly, agitated by this and assume they have done something wrong. 

What the plugin should do is to check the acceptable file types. If there are no filetypes that can be processed by Turnitin it should shut up. It should only complain about types that the student can actually upload.

**Comment by mhughes2k** [comment]
Yes, agree, this has been a bug bear for a long time, with a lot of our students not realising it's just the TII part of the process that is impacted, and *not* the Moodle file submission.
